Gemfile

```diff
@@ -9,3 +9,4 @@ gem 'rake'
 gem 'rspec'
 gem 'rubocop'
 gem 'rubocop-performance'
+gem 'webrick'
```

Comment:
That's for Ruby 3: https://www.ruby-lang.org/en/news/2020/12/25/ruby-3-0-0-released/ (removed from stdlib, need to install separately - looks to be implicitly used by s3 or so)